src/main/java/baseball/PlayBaseballProcess.java

```diff
@@ -0,0 +1,69 @@
+/**
+* @Package_name   : baseball 
+* @Class_name     : FacadeProcess 
+*
+* Create Date : 2023-10-21 
+* Create User : 정은채 
+*/
+package baseball;
+import java.util.ArrayList;
+
+import gamer.ComGamer;
+import gamer.JudgeGamer;
+import gamer.ManageGamer;
+import gamer.UserGamer;
+
+
+public class PlayBaseballProcess {
+
+	// 컴퓨터의 정답 숫자리스트, 사용자의 예상 숫자리스트
+	private ArrayList<Integer> comNumberArr;
+	private ArrayList<Integer> userNumberArr;
```

Comment:
```suggestion
	private List<Integer> comNumberArr;
	private List<Integer> userNumberArr;
```

필드 변수의 타입을 **인터페이스**로 작성하시면 좋을 것 같습니다! 이렇게 하면 나중에 코드를 확장하거나 변경할 때 훨씬 유연하게 대처할 수 있어요. 예를 들어 나중에 `ArrayList`를 사용하다가 다른 리스트 구현체로 변경해야 할 때, 코드의 변경을 최소화할 수 있겠죠?

Reply:
오오 그렇군요!! 참고하겠습니다!